src/web/mjs/engine/Settings.mjs

```diff
@@ -93,6 +93,13 @@ export default class Settings extends EventTarget {
             value: false
         };
 
+        this.retries = {
+            label: 'Retries',
+            description: 'No of times to retry if the site fetch failes',
+            input: types.numeric,
```

Comment:
Restrict range, the ability to set `-312434143` or `3421446354` may confuse the user